src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,33 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post, users, comments }) => {
+  const author = users.find(user => post.userId === user.id);
+  const arrOfComments = comments.filter(
+    comment => comment.postId === post.id,
+  );
```

Comment:
instead of doing it in every component - prepare your list with posts in the App.tsx once (add corresponding properties to each post object) and use it.